src/masternodes/mn_checks.cpp

```diff
@@ -3341,13 +3356,17 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
 
                 if (paybackTokenId == loanTokenId)
                 {
+                    if (loanToken->symbol == "DUSD") {
+                        balances.Sub(tokenAmount);
```

Comment:
why only for DUSD? I know we only need it for DUSD, but would it be a problem to store all loans right away. I think this would be benefical for statistics etc.

Reply:
Hi @kuegi - fair question! also the goal exactly as you indicated to enable more and more statistics on the live namespace.  But currently, the attributes subsystem is pending revamp of internal database. Currently, it's hanging off an implementation that wasn't designed to handle the requirements for such large statistics. So, we're just enabling what's needed for the moment, until attributes are moved to dedicated database, which is in the works.